app/models/agents/phantom_js_cloud_agent.rb

```diff
@@ -0,0 +1,167 @@
+require 'json'
+require 'uri'
+
+include ERB::Util
```

Comment:
This should be included in the `PhantomJsCloudAgent` `class`, not at the top level, because them it gets mixed into everything.